openy.install

```diff
@@ -1041,3 +1041,10 @@ function openy_update_8075() {
     ]);
   }
 }
+
+/**
+ * Enable CKEditor Font module.
+ */
+function openy_update_8076() {
```

Comment:
Please move it to openy_editor.

The module should be enabled prior to the configuration update (https://github.com/ymcatwincities/openy/pull/1717/files#diff-021c3b6b2040e441b3f10fae258fc09dR43)